identity-service/src/models/fingerprint.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+module.exports = (sequelize, DataTypes) => {
+  const Fingerprints = sequelize.define('Fingerprints', {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
```

Comment:
index: true 

or `indexes` https://github.com/AudiusProject/audius-protocol/blob/master/identity-service/src/models/InstagramUser.js#L34

Reply:
not really, but makes it so we can easily see what has indexes if we keep it consistent. we should look at how to generate these from the db state itself